src/group/membership/components/GroupMembershipCard.js

```diff
@@ -146,11 +146,11 @@ const GroupMembershipCard = props => {
   const { groupSlug } = useGroupContext();
   const { onViewMembers, InfoComponent } = props;
   const { fetching, group } = useSelector(
-    _.getOr({}, `group.memberships.${groupSlug}`)
+    state => state.memberships.memberships[groupSlug] ?? {}
```

Comment:
Question: Is there a reason to remove the lodash use here?

Reply:
i generally have been leaning away from lodash because it breaks jump-to-definition/references & can't be typed properly. with typescript, you can define a `useSelector` that's aware of your state type, so by rewriting this my IDE knows what type group is, which made it easier for me to follow the flow in this function and make sure everything was still working.

Reply:
That makes sense to me